### PR TITLE
Implemented: functionality to the Publish segment of the Inventory channels page (#356)

### PR DESCRIPTION
**Pull request by amansinghbais**
### Related Issues
 

 #356

 ### Short Description and Why It's Useful
 
Added functionality in the Publish segment of the inventory channels page.

 ### Screenshots of Visual Changes before/after (If There Are Any)
 

 ### Contribution and Currently Important Rules Acceptance
 

 - [x] I read and followed [contribution rules](https://github.com/hotwax/available-to-promise#contribution-guideline)

### CODE REVIEW

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/components/ShopActionsPopover.vue [hunk 1]
```diff
@@ -23,7 +23,177 @@
...
+  Object.keys(job.runtimeData).map((key: any) => {
+    if(key !== "productStoreId" && key !== "shopifyConfigId" && key !== "shopId") {
+      payload[key] = job.runtimeData[key];
+    }
+  })
+
+  // checking if the runtimeData has productStoreId, and if present then adding it on root level
+  job?.runtimeData?.productStoreId?.length >= 0 && (payload['productStoreId'] = job.status === "SERVICE_PENDING" ? job.productStoreId : store.state.user.currentEComStore.productStoreId)
+  job?.priority && (payload['SERVICE_PRIORITY'] = job.priority.toString())
+  job?.sinceId && (payload['sinceId'] = job.sinceId)
```
Comment (ymaheshwari1) [inline src/components/ShopActionsPopover.vue:166]: Remove this as its not used in this service.

File: src/store/modules/channel/actions.ts [hunk 2]
```diff
@@ -70,6 +71,140 @@ const actions: ActionTree<ChannelState, RootState> = {
     commit(types.CHANNEL_INVENTORY_CHANNELS_UPDATED, groups);
   },
 
+  async fetchShopifyConfigs ({ commit }) {
```
Comment (ymaheshwari1) [inline src/store/modules/channel/actions.ts:74]: commit is defined but not used.

File: src/store/modules/channel/actions.ts [hunk 3]
```diff
@@ -70,6 +71,140 @@ const actions: ActionTree<ChannelState, RootState> = {
...
+      } else {
+        throw resp.data
+      }
+    } catch(error: any) {
+      logger.error(error)
+    }
+    return shopifyConfigs;
+  },
+
+  async fetchJobs ({ commit, dispatch }) {
```
Comment (ymaheshwari1) [inline src/store/modules/channel/actions.ts:99]: Fetch jobs and configs again on changing productStore

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/store/modules/channel/actions.ts (hunk 3 above)
Comment (amansinghbais) [reply src/store/modules/channel/actions.ts:99]: Its already handled in the code.

**Pull request merged**